src/plugins/orangehrmBuzzPlugin/Controller/BuzzMockAPIController.php

```diff
@@ -129,4 +128,55 @@ public function getUpcomingAnniversaries(Request $request): Response
         $response->setStatusCode(Response::HTTP_OK);
         return $response->send();
     }
+
+    /**
+     * @param Request $request
+     * @return Response
+     */
+    public function getPosts(Request $request): Response
+    {
+        $posts = [];
+        $response = new Response();
+        $mockData = array_values(
+            explode(
+                " ",
+                "Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa soluta ullam facilis velit. Voluptatem quisquam unde itaque ipsum natus dolore vero eaque delectus eos. Neque hic totam doloremque itaque nihil"
+            )
+        );
+
+        for ($postCount = 0; $postCount < 50; $postCount++) {
+            array_push($posts, [
```

Comment:
@RajithaKumara this is the post structure i used for mock API. have to discuss further

```
{
  "data": [
    {
      "id": 1,
      "type": "text",
      "text": "Ipsa soluta ullam facilis velit. Voluptatem quisquam unde itaque",
      "employee": {
        "empNumber": 0,
        "employeeId": "000",
        "firstName": "Voluptatem",
        "lastName": "hic",
        "middleName": "amet",
        "terminationId": null
      },
      "stats": {
        "noOfLikes": 82,
        "noOfComments": 24,
        "noOfShares": 45
      },
      "createdTime": 1450260596,
      "updatedtime": 1548444748
    }
  ],
  "meta": {
    "total": 1
  }
}
```